tests/testnet-performance/nightly.yml

```diff
@@ -0,0 +1,77 @@
+steps:
```

Comment:
Can we run all the nightly testing on colo for now?   I think we'll get a ton of benefit out of just using those machines, they're idling overnight and "free".  Once we aren't finding any more issues with colo, we can consider adding a less-regular GCP expensive run or build colo2